app/models/invenio_rdm_record_converter.rb

```diff
@@ -173,9 +173,8 @@ def invenio_metadata
         "creators": @generic_file.creator.map{ |creator| build_creator_contributor_json(creator) },
         "title": @generic_file.title.first,
         "additional_titles": format_additional("title", "alternative-title", @generic_file.title.drop(1)),
-        "description": @generic_file.description.first,
-        "additional_descriptions": format_additional("description", "other", @generic_file.description.drop(1)) + format_additional("description", "acknowledgements", @generic_file.acknowledgments)\
-          + format_additional("description", "abstract", @generic_file.abstract),
+        "description": @generic_file.description.join("\n\n").force_encoding("UTF-8"),
+        "additional_descriptions": format_additional("description", "acknowledgements", @generic_file.acknowledgments) + format_additional("description", "abstract", @generic_file.abstract),
```

Comment:
very minor (i.e. can be merged without) while here to improve readability:

```suggestion
        "additional_descriptions": (
            format_additional(
                "description", "acknowledgements", @generic_file.acknowledgments
            ) +
            format_additional("description", "abstract", @generic_file.abstract)
        ),
```

Reply:
I didn't go for exactly this, but something close